sdk_entrepot_gpf/scripts/entities.py

```diff
@@ -149,7 +149,17 @@ def action_upload_checks(upload: Upload) -> None:
 
     @staticmethod
     def action_upload_delete_files(upload: Upload, delete_files: List[str]) -> None:
+        """
+        Ouvre la livraison puis supprime les fichiers et referme la livraison
+        """
         Config().om.info(f"Suppression de {len(delete_files)} fichiers téléversés sur la livraison {upload} :")
+        upload.api_open()
+        for file in delete_files:
+            if(file.__contains__(".md5")):
```

Comment:
Il faut écrire `".md5" in file` en python, ça va concrètement appeler la fonction __contains__. Là on peut même vérifier que file fini par ".md5" avec :

```suggestion
            if file.endswith(".md5"):
```